docs/docker/README.md

```diff
@@ -1,44 +1,64 @@
 # Minio Docker Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-## Prerequisites
-Docker installed on your machine. Download the relevant installer from [here](https://www.docker.com/community-edition#/download).
+This quickstart guide describes how to quickly install and run a Minio Server Docker container. These are the steps you will follow:
 
-## Run Standalone Minio on Docker.
-Minio needs a persistent volume to store configuration and application data. However, for testing purposes, you can launch Minio by simply passing a directory (`/data` in the example below). This directory gets created in the container filesystem at the time of container start. But all the data is lost after container exits.
+1. [Install Docker](#installdocker) 
+2. [Run Standalone Minio on Docker](#runstandalong) 
+3. [Run Distributed Minio on Docker](#rundistributed) 
+
+## <a name="installdocker"></a>1. Install Docker
+Install Docker using these instructions: <https://www.docker.com/community-edition#/download>.
+
+## <a name="runstandalong"></a>2. Run Standalone Minio on Docker
+
+You can run Minio with either temporary or persistent storage, as described below.
+
+### 2.1 Run Minio with Temporary Storage
+Minio requires a persistent volume to store configuration and application data. However, you can launch Minio by specifying a temporary directory for testing purposes. This temporary directory is created in the container's file system when the container is started, but all of the data is lost after the container exits.
+
+Run the container with a temporary storage directory in `/data`:
 
 ```sh
 docker run -p 9000:9000 minio/minio server /data
```

Comment:
We need to advise that MINIO_ACCESS_KEY and MINIO_SECRET_KEY need to be specified to run in containers as we will not print them in a container.  Alternatively users can add `-it` flag to the docker command but I don't know how common that use case would be.  EDIT - We call this out below, should make mention further up that these are required, even if we just provide an anchor tag to the section below